.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -3081,7 +3081,8 @@ stages:
         DD_API_KEY: $(ddApiKey)
         DD_DOTNET_TRACER_HOME: $(monitoringHome)
 
-    - publish: tracer/build_data/benchmarks
```

Comment:
for my understanding, why are we changing the publish folder? isn't this used later on to generate results on the PR page? I'm thinking of the target `CompareBenchmarksResults` which seems to read from it  in Build.Github.cs🤔

Reply:
Yeah, I was trying to recover the memory dumps from a crash, but I appear to have made a lot whitespace changes and probably a few other mistakes, so I reverted the whole file.